apps/web-portal-client/lib/components/HomePage.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react'
+
+import {Logo} from './Logo'
+import {Card} from './Card'
+import {UserProfile} from './UserProfile'
+import {LoginButton} from './LoginButton'
+
+import {DesktopMenu, MobileMenu, MobileMenuButton} from './main-menu'
+
+export function HomePage() {
+  const [menuStatus, setMenuStatus] = useState(false)
+  const [loginStatus, setLoginStatus] = useState(false)
+
+  return (
+    <div>
+      <nav className="bg-gray-800">
+        <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+          <div className="flex items-center justify-between h-16">
+            <div className="flex items-center">
+              <Logo />
+              <DesktopMenu
+                items={[
+                  {desc: 'Home', link: '#', active: true},
+                  {desc: 'Product', link: '#'},
+                  {desc: 'Teams', link: '#'},
+                  {desc: 'About', link: '#'},
+                ]}
+              />
+            </div>
+            <div className="flex space-x-1">
+              <LoginButton loginStatus={loginStatus} setLoginStatus={setLoginStatus} />
+              <MobileMenuButton menuStatus={menuStatus} setMenuStatus={setMenuStatus} />
+            </div>
```

Comment:
maybe move this section to a separate component that logically joins buttons?

Reply:
To be honest I can't understand sense of this. I simply did conditional rendering depends on toggling button "Login", that's why all components connected with authentication I will try implement carefully in the future